### DISCUSSION TITLE
Any idea of when the ORM will support MSSQL?

### DISCUSSION THREAD

**Discussion by 4parsleythelion** [vlang/v#11811] [Ideas]
I'm having fun playing and learning V, its great! But to use it just about anywhere in my world I need to access Microsoft SQL servers, do you have any idea of when this may happen.

Thank you,

Ashley

**Comment by andrewboldi** [comment]
V already has a MYSQL library.... 
https://github.com/vlang/v/tree/master/vlib/mysql

**Comment by JalonSolov** [comment]
It would take someone who knows MSSQL well enough to write a mssql module for V.

**Comment by spytheman** [comment]
@4parsleythelion are there C drivers/libraries for them?
if yes, then it would be just a question of writing a small V wrapper

**Comment by mvenditto** [comment]
If someone needs a dirty way of accessing SQL Server as of now (or to play around with an ODBC wrapper):
https://github.com/vlang/v/issues/4531#issuecomment-780404301

**Comment by 4parsleythelion** [comment]
Hi all,

Sorry for the delay in getting back to you all, I’ve been dealing with issues. Anyways, if there was a way to use ODBC that would work for a while, but I would love a true / dirty driver to Microsoft SQL Server. How would we go about creating a MSSQL driver, because at some point others will need it?

Thank you all for your kind responses,

Ashley

**Comment by mvenditto** [comment]
Maybe it could be possible to write (or port, looking at something like pyodbc) an odbc driver, and if necessary support MSSQL specific features ontop of it (msodbcsql.h) ?

**Comment by 4parsleythelion** [comment]
I’m more leaning towards pymssql [https://github.com/pymssql/pymssql], but its built on FreeTDS [http://www.freetds.org/], which is not a bad thing, but it would require two ports. But unless I’m missing something we should be able to port pymssql and include FreeTDS.

How much help and guidance would there be if I was going to tackle this monster?

Thanks,

Ashley
